firebase/functions/src/analytics/index.ts

```diff
@@ -0,0 +1,22 @@
+import * as functions from 'firebase-functions'
+import { Request} from 'firebase-functions/lib/providers/https'
+import * as express from "express";
+const fetch = require('node-fetch');
+
+const appKey = functions.config().countly['app_key']
+
+export const uninstall = functions.https.onRequest(async (req: Request, resp: express.Response) => {
+
+    const user = req.query.user
+    const events = JSON.stringify({
+        "key": "Global::uninstallExtension",
+        "count": 1
+    })
+    const urlbase = 'https://analytics.worldbrain.io/i'
+    const urlParams = `${urlbase}?app_key=${appKey}&device_id=${user}&events=${events}`;
+    const url = `${urlbase}?${urlParams}`
+
+    console.log(`Logging uninstall event: ${url}`)
+    await fetch(url)
+    resp.redirect(`https://worldbrain.io/uninstall/?user=${user}`)
```

Comment:
@cdharris this needs to change to simply: https://forms.gle/t9VwLFhx9QYGDbcd6

We don't need the userID at this stage anymore, if I am not mistaken?

Reply:
I'll change it myself though.